content/docs/04-clusters/06-cluster-management/05-backup-restore.md

```diff
@@ -11,320 +11,493 @@ import WarningBox from 'shared/components/WarningBox';
+
+- Google Cloud Platform (GCP) bucket
+
+- MinIO S3 bucket
 
-Palette leverages the BackUps to the following locations:
+- Azure blob storage
 
+While configuring a backup location, you authorize Palette to perform S3 operations in the cloud account. An outstanding use case is when the backup cloud account differs from the one where Palette deploys the clusters. In such scenarios, you must also authorize the latter cloud account, as the diagram below highlights. The numerical points 1, 2, and 3 highlighted in the diagram below signify the following sequence you follow in Palette.
 <br />
 
-#### Amazon Web Services (AWS) S3 Buckets: [Prerequisites](/clusters/cluster-management/backup-restore#foranamazonwebservices(aws)bucketasbackuplocation), [Configure your Backup](/clusters/cluster-management/backup-restore#configureyourbackupinawss3)
+1. When registering a primary cloud account with Palette, you authorize Palette to deploy clusters in the cloud account. 
 
-#### Google Cloud Platform (GCP) Buckets: [Prerequisites](/clusters/cluster-management/backup-restore#foragooglecloudplatform(gcp)backuplocation), [Configure your Backup](/clusters/cluster-management/backup-restore#configureyourbackupingcpbucket)
 
-#### MinIO S3 Buckets: [Prerequisites](/clusters/cluster-management/backup-restore#forminios3backup), [Configure your Backup](/clusters/cluster-management/backup-restore#configureyourbackupinminio)
+2. When adding a secondary cloud account as the backup location, you authorize Palette to store the backup in the cloud account. The secondary cloud account can be the same or different than the primary cloud account. 
 
-#### Azure Blob:[Prerequisites](/clusters/cluster-management/backup-restore#forazureblobbackup),[Configure your Backup](/clusters/cluster-management/backup-restore#configureyourbackupinazure:azureblob)
 
-# Prerequisites
+3. If the secondary cloud account differs from the primary one, you must update the blob storage's access management rules to allow the primary cloud account access to the storage.
 
-## For an Amazon Web Services (AWS) Bucket as Backup Location
 
-* The Amazon Simple Storage Service (S3) permissions listed in the next section need to be configured in the AWS account to provision Backup through Palette.
+![A diagram highlighting a use-case when the backup cloud account is different from the cluster deployment cloud account.](/clusters_cluster-management_backup-restore_separate-cloud-accounts.png)
+
+
+You can establish the authorization or a trust relationship between the cloud entities per the guidelines of your cloud service provider. For example, if you configuring the backup in AWS, you can use Security Token Service (STS), IAM roles, and a trust relationship. In the case of Azure, you can use Shared Access Signatures (SAS), AD roles, and service principal identities. Similarly, you can use service accounts, credentials, and IAM roles for GCP. 
+
+
```

Comment:
We need a title heading here. This is where you start explaining how to create a backup location. Right now there is no transition between the intro and this.